lib/simple_forms_api/form_remediation/configuration/form_0781_config.rb

```diff
@@ -14,8 +36,8 @@ def remediation_data_class
           SimpleFormsApi::FormRemediation::Form0781SubmissionRemediationData
         end
 
-        def temp_directory_path
-          Rails.root.join("tmp/#{SecureRandom.hex}-archive").to_s
```

Comment:
This is [already defined in the Base class](https://github.com/department-of-veterans-affairs/vets-api/blob/019b449f8d18005026e59352927cb9d9ed41e354/lib/simple_forms_api/form_remediation/configuration/base.rb#L50-L55):
```
        # The temporary directory where form submissions will be
        # hydrated and stored. This directory will automatically
        # be deleted once the archive process completes
        def temp_directory_path
          Rails.root.join("tmp/#{SecureRandom.hex}-archive/").to_s
        end
```